src/Multiselect.vue

```diff
@@ -12,7 +12,9 @@
       <slot name="caret" :toggle="toggle">
         <div @mousedown.prevent.stop="toggle()" class="multiselect__select"></div>
       </slot>
-      <slot name="clear" :search="search"></slot>
+      <slot name="clear" :search="search">
+        <div v-if="clearAllButton" class="multiselect__clear" @mousedown.prevent="clearValue">❌</div>
```

Comment:
a `button` element should be used here, rather than a `div`. Semantics are important; if it's a non-semantic `div`, screen readers will not see it. Moreover, the `button` needs to have an accessible name -- on this case "Clear All Values", or similar.